pkg/virt-api/rest/profiler.go

```diff
@@ -117,16 +117,18 @@ func (app *SubresourceAPIApp) getPodsNextPage(cpRequest *v1.ClusterProfilerReque
 }
 
 func podIsReadyComponent(pod *k8sv1.Pod) bool {
+	if pod == nil {
+		return false
+	}
+
 	re, _ := regexp.Compile("^(virt-api-|virt-operator-|virt-handler-|virt-controller-).*")
 	isComponentPod := re.MatchString(pod.Name)
 	// filter out any kubevirt related pod that doesn't have profiling capabilities
 	if !isComponentPod {
 		return false
 	}
 
-	if pod == nil {
-		return false
-	} else if pod.Status.Phase != k8sv1.PodRunning {
+	if pod.Status.Phase != k8sv1.PodRunning {
```

Comment:
move to `switch`?

Reply:
Instead of moving to switch I've simplified this even further as part of [`6b80181` (#13630)](https://github.com/kubevirt/kubevirt/pull/13630/commits/6b8018118b1f858859222036a0a408004bd08153)
PTAL